### PR TITLE
Fixing up login and signup logic and adding alerts

### PR DESCRIPTION
**Pull request by andrewmp1**
Added alerts
- can send event alert w/ a YJ.Alert model
  Fixed up signup/login logic
- can signup and login now but nothing is checking on refreshes to automatically login a user.

### CODE REVIEW

**Comment by oldfartdeveloper** [comment]
Ack! Failed tests!

Seriously, what did you do to get Travis to pay attention to this.  Very cool.

File: app/assets/javascripts/controllers/signup_controller.js.coffee [hunk 1]
```diff
@@ -7,23 +7,29 @@ YJ.SignupController = Em.Controller.extend(
...
+    signup.then ( (data) ->
+      loaded = YJ.store.load(YJ.User, data.user)
+      YJ.set('currentUser', YJ.User.find(loaded.id))
+      Ember.$.cookie('account', data.api_key)
+      self.clearFields()
+      self.transitionToRoute('user.dashboard')
+    ), ->
+      self.set('password', null)
+      alert = YJ.Alert.create(
+        message: "Woops.  An error occured signing up.",
```
Comment (oldfartdeveloper) [inline app/assets/javascripts/controllers/signup_controller.js.coffee]: Suggest: "Whoops!  An error occurred while signing up."

**Comment by oldfartdeveloper** [comment]
I like the changes, thanks.  Noted a couple of typos; make those changes and feel free to merge.

**Comment by andrewmp1** [comment]
Travis now apparently tests any pull requests that are trying to be pulled into a branch they are already running tests against.  I didn't do anything.  They are just THAT good!

**Comment by andrewmp1** [comment]
I'll take a look at this tonight to see if I can't get the tests to pass before I merge this.

**Pull request merged**